fireflyhealth/marketing-website
# feat: add key icon

### Description
Add key icon

### Feedback & Concerns
This should not have any backward-compatible issues. We are adding the icon type, not removing any current icon or anything like that. 

### QA Instructions
Please check the icon locally to make sure it looks as expected. I added the key icon on the staging dataset. 
You can see it here:
- http://localhost:3000/care-team/titus-sanford/
- http://localhost:3000/blog/newsroom/all-article-style/
- http://localhost:3000/test-page/

### Applicable screenshots or Loom video
![image](https://github.com/user-attachments/assets/ad59a5c6-9045-47c2-9355-988f8b327678)
![image](https://github.com/user-attachments/assets/f5c71a8d-f60e-4cb7-9f22-0b2f41687a0f)
![image](https://github.com/user-attachments/assets/d7eb95f1-2f67-4371-9271-d4659a45448f)
![image](https://github.com/user-attachments/assets/f910390c-35d9-4e43-8c9b-a79498a5860b)
![image](https://github.com/user-attachments/assets/89c1ced8-3d85-40c4-b03a-332cb4af30fe)
![image](https://github.com/user-attachments/assets/1362ada3-1568-4ca7-bf28-468cb3ec50d9)
![image](https://github.com/user-attachments/assets/cef269da-a8dc-4f23-b706-4c5cc1d2bdbe)
![image](https://github.com/user-attachments/assets/68f2697e-f7aa-4535-b4db-f90538817ea6)
